R/reprex_inform.R

```diff
@@ -0,0 +1,22 @@
+reprex_quiet <- function() {
+  as.logical(Sys.getenv("REPREX_QUIET", unset = "NA"))
```

Comment:
I went with "environment variable" as opposed to "R option" or "R option and/or environment variable".

Reply:
Is that the standard idiom we use for testing if an envvar is true? I think I usually use something more like `identical(tolower(Sys.getenv("REPREX_QUIET")), "true")`. I don't think it really matters apart from being consistent. Maybe rlang should have a helper that at least reminds us of the standard, even if we're not importing it?

Reply:
I did look around at what we/I do elsewhere while fiddling with this. But I concluded making this truly logical ASAP is quite nice. Never worry again about whether `"true"`, `"True"`, and `"TRUE"` all do the same thing. If we make an effort to be consistent, I would make a case for this way of thinking.

Reply:
The main downside is that allowing `NA` makes the downstream code a bit more complicated

Reply:
I feel like I'm using `NA` for the same purpose that we use a default of `NULL` for function arguments: so I can tell whether user has actually specified the env var (in which case, I honour it) or not (in which case I consult `is_testing()`).